Automattic/jetpack
# [not verified] add instagram widget

WIP 

Fixes #1319

#### Changes proposed in this Pull Request:

* Add a new Instagram widget.

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* pb5gDS-mB-p2

#### Testing instructions:






* Apply D40423-code and sandbox the API.
* Test this widget in both the Customizer and the Widgets page.
* Add the Instagram widget.
* Connect it to your Instagram account.
* Make sure the widget updates to a "connected" state (instead of showing the Connect button, it shows the settings form).
* If in the Customizer, save and publish the changes.
* Check on the front end if the Instagram widget shows up and works as expected.
* Try with several different Instagram widgets.
* Try de-authorizing and re-authorizing the widgets.

#### Proposed changelog entry for your changes:

* Add a new Instagram widget.
